ambry-replication/src/main/java/com/github/ambry/replication/ReplicationMetrics.java

```diff
@@ -671,6 +699,20 @@ public void updateLagMetricForRemoteReplica(RemoteReplicaInfo remoteReplicaInfo,
     }
   }
 
+  /**
+   * Update catch up point of local replica from the cloud replica.
+   * @param remoteReplicaInfo {@link RemoteReplicaInfo} of the cloud replica.
+   * @param catchUpPoint timestamp upto which local replica has caught with the cloud replica.
+   */
+  public void updateCatchupPointMetricForRemoteReplica(RemoteReplicaInfo remoteReplicaInfo, long catchUpPoint) {
+    // update this metric only for cloud peer replica. There will only be one cloud replica peer per partition.
+    if (remoteReplicaInfo.getReplicaId().getReplicaType() == ReplicaType.CLOUD_BACKED
```

Comment:
If this is only for cloud replicas, should the metric name reflect that?

Reply:
done.